docs/sources/reference/templating.md

```diff
@@ -114,6 +114,33 @@ String to interpolate: '${servers:percentencode}'
 Interpolation result: 'foo%28%29bar%20BAZ%2Ctest2'
 ```
 
+### Singlequote
+Formats single- and multi-valued variables into a comma-separated string, quotes each value with `'`.
+
+```bash
+servers = ['test1', 'test2']
+String to interpolate: '${servers:singlequote}'
+Interpolation result: "'test1','test2'"
+```
+
+### Doublequote
+Formats single- and multi-valued variables into a comma-separated string, quotes each value with `"`.
+
+```bash
+servers = ['test1', 'test2']
+String to interpolate: '${servers:doublequote}'
+Interpolation result: '"test1","test2"'
+```
```

Comment:
without a way to escape the values (single or double quotes), won't that limit the usefulness of these?

Reply:
yes, I added the escape logic.